packages/access-api/src/bindings.d.ts

```diff
@@ -71,6 +72,8 @@ export interface RouteContext {
     validations: Validations
   }
   uploadApi: ConnectionView
+  uploadApiUrl: URL
```

Comment:
I know these are one and the same but we should confiugure it has a separate service here.

```suggestion
  ucanInvocationPostURL: URL
```